src/objects/core/zcl_abapgit_serialize.clas.abap

```diff
@@ -504,11 +508,11 @@ CLASS zcl_abapgit_serialize IMPLEMENTATION.
     IF mi_log IS BOUND.
       LOOP AT lt_unsupported_count ASSIGNING <ls_unsupported_count>.
         IF <ls_unsupported_count>-count = 1.
-          mi_log->add_error( iv_msg  = |Object type { <ls_unsupported_count>-obj_type } not supported, {
-                                       <ls_unsupported_count>-obj_name } ignored| ).
+          mi_log->add_error( |Object type { <ls_unsupported_count>-obj_type } not supported, | &&
+                             |<ls_unsupported_count>-obj_name } ignored| ).
```

Comment:
Hi, seems like this line: |<ls_unsupported_count>-obj_name } ignored| ). is missing a starting {.
It should be: |{ <ls_unsupported_count>-obj_name } ignored| ).

Reply:
https://github.com/abapGit/abapGit/pull/6528